config-scripts/cups-defaults.m4

```diff
@@ -396,15 +396,17 @@ AS_IF([test $CUPS_WEBIF = Yes || test $CUPS_BROWSING = Yes], [
 AC_SUBST([SYSTEMD_WANTED_BY])
 
 dnl Set default value of IdleExitTimeout
-AC_ARG_WITH([idle_exit_timeout], AS_HELP_STRING([--with-idle-exit-timeout], [set the default value for IdleExitTimeout, default=60]), [
-    AS_IF([test "x$withval" = "xno"], [
-	EXIT_TIMEOUT=0
+AS_IF([test x$with_ondemand != xno], [
+    AC_ARG_WITH([idle_exit_timeout], AS_HELP_STRING([--with-idle-exit-timeout], [set the default value for IdleExitTimeout, default=60]), [
```

Comment:
Don't we need to set `EXIT_TIMEOUT` to the empty string if `--without-ondemand` is specified?

Reply:
@michaelrsweet I have tested it locally, and `--without-ondemand` worked as I expected - there was no `IdleExitTimeout` line in cupsd.conf.

Tbh I would not leave any comment about IdleExitTimeout if ondemand is disabled - IMO person who builds CUPS without ondemand support on purpose won't be interested in a comment about ondemand specific conf directive. If you don't mind, I will only set `EXIT_TIMEOUT` to empty string, if `with_ondemand = no` for better code style and if on other archs/OS behaves configure differently.